WcaOnRails/app/controllers/api/internal/v1/api_controller.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 class Api::Internal::V1::ApiController < ApplicationController
-  prepend_before_action :validate_token
+  # DON'T APPROVE WITH THIS CHANGE
+  # prepend_before_action :validate_token
```

Comment:
@FinnIckler @dunkOnIT: I like to use the permissions API for enabling or disabling edit icon in roles tab. But I can't use it directly because it requires some headers. To make it work, I commented this line. Pretty sure this is not the correct solution, could you please let me know what has to be done for me to use it in our website?

Reply:
just use the non internal route api/v0/users/me/permissions

Reply:
Done, thank you :-)